src/main/java/com/google/sps/utility/KeyProvider.java

```diff
@@ -29,7 +29,11 @@
 
   /** Constructor to create instance with file linked to default path. */
   public KeyProvider() {
-    this.file = new File("src/main/resources/KEYS.json");
+    if (System.getProperty("user.dir").contains("target")) {
+      this.file = new File("../classes/KEYS.json");
+    } else {
+      this.file = new File("src/main/resources/KEYS.json");
+    }
```

Comment:
I don't think this was needed before. Not sure why KEYS.json is moved to classes instead of the predetermined src/main/resources. I'll have to clarify with @gbuenoandrade - however, this works.

Reply:
Should this go into a separate change? It seems unrelated to the rest of the changes in this PR.

(If you need this change for the rest of the PR to work, consider merging it into master first - should be relatively quick to review.)

Reply:
Maybe you ran `mvn package` from a different directory? (https://stackoverflow.com/a/16239152/13744548) (I'm not sure if it's related.)

In any case, I think we can move this to a different PR, especially if it is currently working when deployed by GitHub.

Reply:
@antmar I agree, I'll be sure to abstract this into another PR. Gui's point makes sense too, this change is just to have it working on my local machine.
@gbuenoandrade When I was debugging this I found that `mvn package` was run from `/src/target/step7-2020-1` which I could be wrong but I think is expected(?) however what I thought was weird is `KEYS.json` gets put into the `/src/target/classes` file instead of `/src/target/step7-2020-1/resources/` or something similar.